### ISSUE TITLE
Update to new 4.4.1 Version

### ISSUE THREAD

**Issue by THS-on** [flathub/org.musescore.MuseScore#132]
There is a new version of MuseScore. One of the major changes is the update to QT 6.2.4. This seems to make it a bit more difficult to update the Flatpak, as it does not build against the EOL 6.2 runtime which errors with:
```
-- Found WrapVulkanHeaders: /usr/include  
CMake Error at buildscripts/cmake/FindQt6.cmake:50 (find_package):
  By not providing "FindQt6Core5Compat.cmake" in CMAKE_MODULE_PATH this
  project has asked CMake to find a package configuration file provided by
  "Qt6Core5Compat", but CMake did not find one.

  Could not find a package configuration file provided by "Qt6Core5Compat"
  with any of the following names:

    Qt6Core5CompatConfig.cmake
    qt6core5compat-config.cmake

  Add the installation prefix of "Qt6Core5Compat" to CMAKE_PREFIX_PATH or set
  "Qt6Core5Compat_DIR" to a directory containing one of the above files.  If
  "Qt6Core5Compat" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  CMakeLists.txt:155 (include)
```

Updating to QT 6.5+ seems to have other issues: https://github.com/musescore/MuseScore/issues/24097

What is the best way for us to update the package?

**Comment by hfiguiere** [comment]
it fail to build for other reason here anyway.

The best way? Fix upstream.

**Comment by hfiguiere** [comment]
note that I wouldn't accept a PR to use an EOL runtime and I think the qt5 compat is in later versions

**Comment by hfiguiere** [comment]
```
-- Configuring freetype <>
CMake Error at src/framework/draw/cmake/SetupHarfBuzz.cmake:53 (harfbuzz_Populate):
  Unknown CMake command "harfbuzz_Populate".
Call Stack (most recent call first):
  src/framework/draw/cmake/SetupHarfBuzz.cmake:53 (cmake_language)
  src/framework/draw/CMakeLists.txt:110 (include)


-- Configuring incomplete, errors occurred!
```

that's the current error and haven't time to dig what dependency is missing. but I suspect this will lead to trying to download something which is a no-no.

**Comment by THS-on** [comment]
> note that I wouldn't accept a PR to use an EOL runtime and I think the qt5 compat is in later versions

That makes sense. We then probably need to wait on upstream to be at least compatible with QT 6.6.

**Comment by hfiguiere** [comment]
it's like waiting for Godot. I wouldn't hold my breath. I haven't found MuseScore developer proactive in that kind of fixes.

**Comment by THS-on** [comment]
> that's the current error and haven't time to dig what dependency is missing. but I suspect this will lead to trying to download something which is a no-no.

That issue can be probably fixed by using `-D MUE_COMPILE_USE_SYSTEM_HARFBUZZ=ON` (https://github.com/musescore/MuseScore/commit/696279e362afe72db5e92f8a47aa64b3a0e86a86) and it seems to build then against 6.6.

I've put my current changes here: https://github.com/THS-on/org.musescore.MuseScore/tree/4.1.1

> it's like waiting for Godot. I wouldn't hold my breath. I haven't found MuseScore developer proactive in that kind of fixes.

Luckily for that issue there seems to be a patch: https://github.com/musescore/MuseScore/pull/24326

**Comment by THS-on** [comment]
The flatpak builds, but crashes with:
```
QGuiApplication::setDesktopFileName: the specified desktop file name ends with .desktop. For compatibility reasons, the .desktop suffix will be removed. Please specify a desktop file name without .desktop suffix
Detected locale "C" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.
Qt: Session management error: Could not open network socket
qt.qml.typeregistration: Invalid QML element name "IconCode"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "MusicalSymbolCodes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "ContainerType"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "NavigationEvent"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "MUAccessible"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "CompareType"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "SelectionMode"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "ToolBarItemType"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "DockToolBarAlignment"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "Location"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "CloudVisibility"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "SaveToCloudResponse"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "DirectionTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "SlurTieTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "NoteHead"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "Beam"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "Hairpin"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "OrnamentTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "Dynamic"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "Glissando"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "CommonTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "BarlineTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "MarkerTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "KeySignatureTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "AccidentalTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "FretDiagramTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "LineTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "TextTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "ArticulationTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "AmbitusTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "ChordSymbolTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "BendTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "TremoloBarTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "TremoloTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "VoiceTypes"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "InstrumentsTreeItemType"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "StandardButton"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "SaveLocationType"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "GenerateAudioTimePeriodType"; value type names should begin with a lowercase letter
qt.qml.typeregistration: Invalid QML element name "MigrationType"; value type names should begin with a lowercase letter
20:37:37.550 | INFO  | main_thread     | GlobalModule::onPreInit | log path: /home/user/.var/app/org.musescore.MuseScore/data/MuseScore/MuseScore4/logs/MuseScore_240906_203737.log
20:37:37.550 | INFO  | main_thread     | GlobalModule::onPreInit | === Started MuseScore Studio 4.4.1, build:  ===
20:37:37.551 | INFO  | main_thread     | LanguagesService::effectiveLanguageCode | System language code: "C" 
20:37:37.551 | ERROR | main_thread     | LanguagesService::setCurrentLanguage | Unsupported language code: "system" 
20:37:37.551 | WARN  | main_thread     | IpcSocket::connect | failed connect to server
20:37:37.551 | WARN  | main_thread     | IpcSocket::connect | failed connect to server
20:37:37.551 | INFO  | main_thread     | IpcSocket::connect | success connected to ipc server
20:37:37.551 | INFO  | 138216768276160 | IpcServer::listen | id: "06cbfce48a4247e7af22f650ba7265c3" 
20:37:37.554 | WARN  | main_thread     | DiagnosticsModule::onInit | crash handling disabled
20:37:37.591 | ERROR | main_thread     | MuseSamplerLibHandler::MuseSamplerLibHandler | Unable to open MuseSampler library, path: /home/user/.var/app/org.musescore.MuseScore/data/MuseSampler/lib/libMuseSamplerCoreLib.so
20:37:37.591 | ERROR | main_thread     | MuseSamplerResolver::doInit | Incompatible MuseSampler library; ignoring
20:37:37.591 | ERROR | main_thread     | MuseSamplerLibHandler::MuseSamplerLibHandler | Unable to open MuseSampler library, path: libMuseSamplerCoreLib.so
20:37:37.591 | ERROR | main_thread     | MuseSamplerResolver::doInit | Incompatible MuseSampler library; ignoring
20:37:37.592 | WARN  | main_thread     | Qt              | QIODevice::read (QFile, "/home/user/.var/app/org.musescore.MuseScore/data/MuseScore/MuseScore4/shortcuts.xml"): device not open
20:37:37.592 | WARN  | main_thread     | Qt              | QIODevice::read (QFile, "/home/user/.var/app/org.musescore.MuseScore/data/MuseScore/MuseScore4/midi_mappings.xml"): device not open
20:37:37.594 | WARN  | main_thread     | AbstractCloudService::readTokens | Could not find the tokens file: /home/user/.var/app/org.musescore.MuseScore/data/MuseScore/MuseScore4/musescorecom_cred.dat
20:37:37.594 | WARN  | main_thread     | AbstractCloudService::readTokens | Could not find the tokens file: /home/user/.var/app/org.musescore.MuseScore/data/MuseScore/MuseScore4/audiocom_cred.dat
20:37:37.843 | INFO  | main_thread     | GuiApp::perform | Using graphics api: "opengl" 
20:37:37.843 | INFO  | main_thread     | GuiApp::perform | Detecting problems with graphics api
20:37:38.037 | WARN  | main_thread     | Qt              | QQmlApplicationEngine failed to load component
20:37:38.037 | ERROR | main_thread     | GuiApp::perform | error: qrc:/qml/platform/linux/Main.qml:57:5: Type WindowContent unavailable

20:37:38.037 | ERROR | main_thread     | GuiApp::perform | error: qrc:/qml/WindowContent.qml:97:9: Type NotationPage unavailable

20:37:38.037 | ERROR | main_thread     | GuiApp::perform | error: qrc:/qml/NotationPage/NotationPage.qml:246:13: Type InstrumentsPanel unavailable

20:37:38.037 | ERROR | main_thread     | GuiApp::perform | error: qrc:/qml/MuseScore/InstrumentsScene/InstrumentsPanel.qml:234:25: Cannot override FINAL property

20:37:38.037 | WARN  | main_thread     | Qt              | qrc:/qml/platform/linux/Main.qml:57:5: Type WindowContent unavailable
20:37:38.037 | WARN  | main_thread     | Qt              | qrc:/qml/WindowContent.qml:97:9: Type NotationPage unavailable
20:37:38.037 | WARN  | main_thread     | Qt              | qrc:/qml/NotationPage/NotationPage.qml:246:13: Type InstrumentsPanel unavailable
20:37:38.037 | WARN  | main_thread     | Qt              | qrc:/qml/MuseScore/InstrumentsScene/InstrumentsPanel.qml:234:25: Cannot override FINAL property
20:37:38.038 | ERROR | main_thread     | GuiApp::perform | failed Qml load
```

**Comment by hfiguiere** [comment]
closed by #133
